### PR TITLE
Add intention setting fields to signup form

### LINKED ISSUE #93: Add fields for intention category and focus to the sign up form
In the second part of the sign up form (www.shinetext.com/sms-settings), add two fields for people to set their intentions.

- Intention category: a dropdown select field where people can only select one option
- Intention focus: a short single line text field
- Should update values on both our own database and on users' SMS profiles

### PR DESCRIPTION
**Pull request by cl2205**
#### What's this PR do?
- Adds 2 new form fields for intention setting (category & focus) on 2nd part of sign up form.
- Applies prettier.js code formatting and adds Github PR template to repo

#### How was this/how should this be reviewed?
- Tested manually in local env using node-inspector and browser. Will want to check that
1.) that the user profile is being updated in Mobile Commons with intention info (is there a dev 
2.) our user db is getting the intention data from MC within 10 mins 
(assuming MC and lightbeam is set up to support our pre-prod sign ups?)

#### What are the relevant tickets?
Closes #93 

#### Questions/Considerations
See code comments

### CODE REVIEW

File: views/sms-settings.ejs [hunk 1]
```diff
@@ -90,6 +90,32 @@
...
     </fieldset>
+    
+    <fieldset>
+      <div class="label-container">
+        <label>What's one thing you want to work on most?</label>
+        <p>Focusing on ONE area is key.</p>
+      </div>
+      <div class="input-container">
+        <select id="intention-category" name="intention-category">
+          <option value="presence" selected="selected">Joy & Presence</option>
```
Comment (cl2205) [inline views/sms-settings.ejs]: these option values are currently set to match the string values extracted in lightbeam logic:
https://github.com/shinetext/lightbeam/blob/dev/tasks/mc-profiles.js#L190

File: views/sms-settings.ejs [hunk 2]
```diff
@@ -90,6 +90,32 @@
         <select id="bday-year" name="bday-year"></select>
       </div>
     </fieldset>
+    
+    <fieldset>
+      <div class="label-container">
+        <label>What's one thing you want to work on most?</label>
```
Comment (cl2205) [inline views/sms-settings.ejs]: The copy for the labels and menu options pretty much come from the copy currently being used for intention-setting flow over SMS.

**Comment by cl2205** [comment]
**Desktop/Laptop View:**
desktop_view

**Mobile View:**
mobile_view

**Comment by jonuy** [comment]
Assigning this to myself since it's currently on us to get the messaging in place before this is something we can merge in.

**Comment by jonuy** [comment]
We don't have any intention (no pun intended) of rolling this out right now, so I'm gonna close for now.

**Pull request closed without merging**